graphqlapi/pom.xml

```
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <!-- inherit common settings -->
+  <parent>
+    <groupId>io.stargate</groupId>
+    <artifactId>stargate</artifactId>
+    <version>0.0.6-SNAPSHOT</version>
+  </parent>
+  <groupId>io.stargate.graphql</groupId>
+  <artifactId>graphqlapi</artifactId>
+  <properties>
+    <osgi.bundle.version>${project.version}</osgi.bundle.version>
+    <jetty.version>9.4.30.v20200611</jetty.version>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>io.stargate.db</groupId>
+      <artifactId>persistence-api</artifactId>
+      <version>0.0.5-SNAPSHOT</version>
```

This module is depending on an older snapshot of `persistence-api` and `authentication`
```suggestion
      <version>${project.version}</version>
```

---

Good catch, addressing this as I'm merging the latest changes from the base branch.